packages/pds/src/crawlers.ts

```diff
@@ -18,20 +23,22 @@ export class Crawlers {
       return
     }
 
-    await Promise.all(
-      this.agents.map(async (agent) => {
-        try {
-          await agent.api.com.atproto.sync.requestCrawl({
-            hostname: this.hostname,
-          })
-        } catch (err) {
-          log.warn(
-            { err, cralwer: agent.service.toString() },
-            'failed to request crawl',
-          )
-        }
-      }),
-    )
-    this.lastNotified = now
+    await this.backgroundQueue.add(async () => {
+      await Promise.all(
+        this.agents.map(async (agent) => {
```

Comment:
I notice the background queue task is awaited here, so it's not backgrounded relative to `notifyOfUpdate()`.  Is the purpose of putting this on the queue to limit concurrency or something along those lines?

Reply:
ah hmmm yeah this should probably not be awaited 👍